etc/dolphin/dolphin.conf

```diff
@@ -5,3 +5,10 @@ dolphin_cryptor = dolphin.cryptor._Base64
 [database]
 connection = sqlite:////var/lib/dolphin/dolphin.sqlite
 db_backend = sqlalchemy
+
+[redis]
+db_name = <db_name>
+user = <db_user>
+password = <db_password>
```

Comment:
It's not safe to save password in config file.

Reply:
Password in this config file need to be encrypted, and the code would decrypt it then use it.